doc/nrf/config_and_build/configuring_app/index.rst

```diff
@@ -53,9 +55,12 @@ Just as for creating the application, you can build the application using either
          See :ref:`programming_board_names` for more information on the supported boards and board targets.
          The board targets supported for a given application are always listed in its requirements section.
 
-         After running the ``west build`` command, the build files can be found in :file:`build/zephyr`.
-         |output_files_note|
-         For more information on the contents of the build directory, see :ref:`zephyr:build-directory-contents` in the Zephyr documentation.
+         In the |NCS|, this command builds the application with :ref:`sysbuild enabled by default <sysbuild_enabled_ncs>`.
+         It can be expanded with :ref:`optional_build_parameters`, such as :ref:`custom CMake options <cmake_options>` or the ``--no-sysbuild`` parameter that disables building with sysbuild.
+
+       After running the ``west build`` command, the build files can be found in :file:`build/zephyr` by default.
```

Comment:
this is not correct, see https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-15845/nrf/config_and_build/sysbuild_migration.html